src/status_im/contexts/wallet/send/input_amount/component_spec.cljs

```diff
@@ -100,60 +100,4 @@
           (.then (fn []
                    (h/is-truthy (h/get-by-label-text :button-one))
                    (h/fire-event :press (h/get-by-label-text :button-one))
-                   (h/was-called on-confirm))))))
-
-  (h/test "Try to fill more than limit"
-    (h/setup-subs sub-mocks)
-    (h/render [input-amount/view
-               {:crypto-decimals 10
-                :limit-crypto    286}])
-
-    (h/fire-event :press (h/query-by-label-text :keyboard-key-2))
-    (h/fire-event :press (h/query-by-label-text :keyboard-key-9))
-    (h/fire-event :press (h/query-by-label-text :keyboard-key-5))
-
-    (-> (h/wait-for #(h/is-truthy (h/get-by-text "$290.00")))
-        (.then (fn []
-                 (h/fire-event :press (h/query-by-label-text :keyboard-key-backspace))
-                 (h/fire-event :press (h/query-by-label-text :keyboard-key-8))
-                 (h/fire-event :press (h/query-by-label-text :keyboard-key-5))
-                 (h/wait-for #(h/get-by-text "$2850.00"))))))
-
-  (h/test "Try to fill more than limit"
```

Comment:
this test should still work but the check should be updated 👍 .
we can technically fill more than limit no? 🤔

Reply:
The same thing here